src/components/Home/Header.module.css

```diff
@@ -22,7 +22,12 @@
 
 .title,
 .body {
-  color: var(--font-color-text-inverted);
+  color: var(--sphereon-white-gray);
+}
+
+.title {
+  font-weight: var(--font-weight-bolder);
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.14);
```

Comment:
When i load up the portal i can see there is a box shadow arround: Solving mobility challenges.

is this ment to be there? it looks kinda weird

Reply:
Removed!